sklearn/calibration.py

```diff
@@ -89,6 +89,11 @@ class CalibratedClassifierCV(BaseEstimator, ClassifierMixin,
         .. versionchanged:: 0.22
             ``cv`` default value if None changed from 3-fold to 5-fold.
 
+    strict : str, default=True
+        For the isotonic method, if set to True, this impose strict
+        monotonicity constraints in order to preserve rank-based metrics
+        after calibration.
```

Comment:
```suggestion
        For the isotonic method, if set to True, this imposes strict
        monotonicity constraints. Ignored if `method='sigmoid'`.
```

I think we should mention the effect on rank-based metrics in the User Guide instead